packages/ffe-file-upload/less/file-upload.less

```diff
@@ -0,0 +1,337 @@
+@ffe-blue-royal-escape: escape('@{ffe-blue-royal}');
+@ffe-blue-royal-fill: @ffe-blue-royal-escape;
+@ffe-blue-azure-escape: escape('@{ffe-blue-azure}');
+@ffe-blue-azure-fill: @ffe-blue-azure-escape;
+
+.ffe-file-upload {
+    margin-top: 45px;
+    display: -ms-grid;
```

Comment:
Vendor prefixes are added at build time, so no need to include them in the Less files (will also cause stylelint errors/warnings).

Reply:
Should be fixed now (otherwise Travis will complain anyway :smile:  )